presto-druid/src/test/java/io/prestosql/plugin/druid/DruidIntegrationSmokeTest.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.druid;
+
+import io.prestosql.testing.AbstractTestIntegrationSmokeTest;
+import io.prestosql.testing.QueryRunner;
+import org.intellij.lang.annotations.Language;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static io.prestosql.tpch.TpchTable.ORDERS;
+
+@Test
+public class DruidIntegrationSmokeTest
+        extends AbstractTestIntegrationSmokeTest
+{
+    private TestingDruidServer druidServer;
+
+    public DruidIntegrationSmokeTest()
+    {
+        try {
+            Path storage = Paths.get("storage");
+            if (Files.exists(storage)) {
+                druidServer = new TestingDruidServer(storage.toAbsolutePath().toString());
+            }
+            else {
+                druidServer = new TestingDruidServer(Files.createDirectory(storage).toAbsolutePath().toString());
```

Comment:
move IO and instnatiation of the server to `createQueryRunner` method.
Test constructors should be as simple as possible, because they are run eagerly at the very start of all tests.
Also, the error reporting for ctor failures is suboptimal.